src/Microsoft.Extensions.DependencyInjection.Specification.Tests/Fakes/FakeService.cs

```diff
@@ -2,9 +2,61 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 
 namespace Microsoft.Extensions.DependencyInjection.Specification.Fakes
 {
+    public class FakeDisposeCallback
```

Comment:
There's already 1000 .cs files in Specification.Tests/Fakes, just put all the public classes in their own file.